shared/oval/root_path_no_dot.xml

```diff
@@ -3,7 +3,7 @@
     <metadata>
       <title>Ensure that No Dangerous Directories Exist in Root's Path</title>
```

Comment:
This one seems to be passing anytime (even if ```~/.bashrc``` contains something like:
```
PATH=$PATH:
```
But this is problem of RHEL-6 implementation => better to be handled within separate PR).